.github/workflows/run-snapshot-generation.yml

```diff
@@ -0,0 +1,103 @@
+---
+name: Snapshot recording
+
+on:
+  pull_request:
+    types: [ opened, edited , synchronize ]
+
+permissions:
+  checks: write
+  contents: write
+  statuses: write
+  pull-requests: write
+  actions: write
+
+jobs:
+  generate_snapshots:
+    if: ${{contains(github.event.pull_request.title, '[New snapshots]')}}
+    name: Generate snapshots
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          lfs: true
+
+      - name: Set up Git LFS
+        run: |
+          git lfs install
+
+      - name: Gradle - Record snapshots with Paparazzi
+        id: testStep
+        run: ./gradlew clean components:recordPaparazziDebug
```

Comment:
question: upcoming tests has added `if: always()`, do we need this? I think the gradle command should succeed if new screenshots are recorded correctly?

Reply:
Hmm. I think I don't understand. What do you mean by upcoming tests? Do you mean the steps after recording - with comment updating?

Reply:
Indeed, I thought about "steps". I think `Gradle - Record snapshots with Paparazzi` step should always succeed. Even if new screenshots are generated. So if this step always succeeds, we don't need to add `if: failure()` and `if: always()` to steps that follow `Gradle - Record snapshots with Paparazzi` step. Am I right?

Reply:
Actually it is possible that recording fails. For example, upgrading the ktx version to 1.10.0 make recording fail. That's a paparazzi bug. So if one bug exists and can fail the job then I believe it is helpful to have that.